tests/php/Controllers/Root/RootController/RootControllerDocsActionTest.php

```diff
@@ -443,4 +443,21 @@ public function testDocsActionIsTabbarStatusEqualHomeWhenIsDevParam()
             'error:アサイン情報としてtabbarStatusはdocsではない'
         );
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)